smart-cards/src/main/java/gov/va/api/health/smartcards/patient/ImmunizationTransformer.java

```diff
@@ -16,18 +21,44 @@ static Reference referenceOnly(@NonNull Reference reference) {
     return Reference.builder().reference(reference.reference()).build();
   }
 
-  Reference location() {
-    if (entry.resource().location() == null) {
-      return null;
+  String occurrenceDateTime() {
+    // only publish the date information. time is not necessary.
+    String odt = entry.resource().occurrenceDateTime();
+    // expect DQ to provide a ISO8601-formatted date
+    if (odt.contains("T")) {
+      odt = Iterables.get(Splitter.on('T').split(odt), 0);
     }
-    return referenceOnly(entry.resource().location());
+    return odt;
   }
 
   Reference patient() {
     // Do not include display
     return referenceOnly(entry.resource().patient());
   }
 
+  List<Immunization.Performer> performer() {
+    if (CollectionUtils.isEmpty(entry.resource().contained())) {
+      return null;
+    }
+    Location location =
+        entry.resource().contained().stream()
+            .filter(r -> r instanceof Location)
+            .map(r -> (Location) r)
+            .findFirst()
+            .orElse(null);
+    if (location == null) {
+      return null;
+    }
+    Reference org = location.managingOrganization();
+    if (org == null || StringUtils.isEmpty(org.display())) {
```

Comment:
perhaps we should bring over the `isBlank` method from DQ? That can replace `StringUtils.isEmpty` and `CollectionUtils.isEmpty`

Reply:
There'd be too few usages of it so I'd say we should defer this until it can be reused outside of this class? There's only a single reference to `StringUtils.isEmpty` in `PatientController`. Is that enough to generalize the isBlank?